packages/react/src/components/Chat/ChatItem.tsx

```diff
@@ -46,19 +54,22 @@ class ChatItem extends UIComponent<ReactProps<ChatItemProps>, any> {
 
   static propTypes = {
     ...commonPropTypes.createCommon({ content: false }),
+    accessibility: customPropTypes.accessibility,
```

Comment:
I would definitely expect this to be in the commonPropTypes.createCommon factory.. We can use {accessibility: false} if we don't want it there for some components

Reply:
I like `createCommon()`, but https://github.com/stardust-ui/react/pull/647#discussion_r247304323

Reply:
Ok, if we agree to remove this abstraction, then we should do this, but having now some property being defined directly, and some of them as result of a function is really confusing. Let's create then issue for this, because I missed that comment and was not aware of it by now...